static/locales/en-US.yaml

```diff
@@ -470,6 +470,15 @@ Settings:
     Export History: Export History
     Import Playlists: Import Playlists
     Export Playlists: Export Playlists
+    Export Playlists For Older FreeTube Versions:
+      Label: Export Playlists For Older FreeTube Versions
+      # |- = Keep newlines, No newline at end
+      Tooltip: |-
+        This option exports videos from all playlists into one playlist named `Favorites`.
```

Comment:
**nitpick:** use quotes, not backticks

Reply:
Done
![image](https://github.com/PikachuEXE/FreeTube/assets/1018543/89a5acd1-6e54-4243-9f4b-4d93bae84c0b)